src/pincode/authentication.test.ts

```diff
@@ -649,3 +650,20 @@ describe(checkPin, () => {
     expect(mockStore.dispatch).not.toHaveBeenCalled()
   })
 })
+
+describe(_getPasswordHash, () => {
+  // See some these are producing the same hash
+  // Because the current implementation treats the input as hex string (without the '0x' prefix)
+  // But we'll change this to treat the input as a raw string/buffer in the future
+  it.each([
+    ['123456', 'bf7cbe09d71a1bcc373ab9a764917f730a6ed951ffa1a7399b7abd8f8fd73cb4'],
+    ['0x123456', 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'],
+    ['0x1234567', 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'],
+    ['abx', '087d80f7f182dd44f184aa86ca34488853ebcc04f0c60d5294919a466b463831'],
+    ['abxy', '087d80f7f182dd44f184aa86ca34488853ebcc04f0c60d5294919a466b463831'],
+    ['ABXY', '087d80f7f182dd44f184aa86ca34488853ebcc04f0c60d5294919a466b463831'],
+    ['0xabxy', 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'],
```

Comment:
These hashes were captured using the implementation before switching to viem.
So we keep the same behavior.